tests/models/test_taskinstance.py

```diff
@@ -119,6 +125,18 @@ def setUp(self):
     def tearDown(self):
         self.clean_db()
 
+    def test_load_error_file_returns_None_for_closed_file(self):
+        error_fd = create_tempfile()
```

Comment:
It’s probably easier (and slightly more efficient) to do `io.TextIO()` instead.

Reply:
In my opinion, using a temp file will help us use the method `set_error_file` line136, thereby mimicking the actual run. What do you think?

Reply:
Yeah, not sure `open()` will work on `BytesIO()`?